dll/apisets/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ function (add_apiset apiset_name baseaddress)
         ${CMAKE_CURRENT_BINARY_DIR}/${apiset_name}.def)
 
     add_dependencies(${apiset_name} xdk)
-    set_module_type(${apiset_name} win32dll ENTRYPOINT 0 UNICODE IMAGEBASE ${baseaddress})
+    set_module_type(${apiset_name} module UNICODE IMAGEBASE ${baseaddress})
```

Comment:
While I totally agree with this change, this doesn't actually address the real issue (which is a broken CRT). I have an additional fix for the CRT ready soon.